apps/i18n/common/en_us.json

```diff
@@ -1169,6 +1172,8 @@
   "manageStudentsNotificationAtCapacity": "You can’t have more than 500 students in a section. Questions about this limit? [Contact support](https://code.org/contact)",
   "manageStudentsNotificationAddSuccess": "You added {numStudents, plural, one {1 student} other {# students}} to your section.",
   "manageStudentsNotificationSuccess": "Success!",
+  "manageStudentsNotificationLocked": "You have locked this section",
+  "manageStudentsNotificationLockedDetails": "Students can not join this section using the section code. {loginType, select, email {You can still add students to the section manually by moving or copying them from another section} other {You can still add students manually on the Manage Students tab}}. You can unlock this section by clicking \"Edit Section Details\".",
```

Comment:
Nit: "can not" should be one word: "cannot"

Reply:
Thank you! Just pushed a fix for that typo